src/data/contexts/alertsPromptContext.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { createContext, useContext, useState, ReactNode, useEffect } from 'react';
+import './alertsPromptContext.scss';
+import { CheckCircle, Info, Warning, XCircle } from '@phosphor-icons/react';
+
+const AlertsContext = createContext(undefined);
```

Comment:
A context initializer should be provided with a `<T>` type and default properties as fallback.